src/ensembl/src/content/app/custom-download/CustomDownload.tsx

```diff
@@ -1,22 +1,45 @@
-import React, { FunctionComponent } from 'react';
+import React, { FunctionComponent, useEffect } from 'react';
 import { connect } from 'react-redux';
 import PreFilterPanel from 'src/content/app/custom-download/containers/pre-filters-panel/PreFilterPanel';
-import { getShowPreFilterPanel } from './state/customDownloadSelectors';
+import {
+  getShowPreFilterPanel,
+  getCustomDownloadActiveGenomeId
+} from './state/customDownloadSelectors';
 import { RootState } from 'src/store';
 import CustomDownloadHeader from './containers/header/CustomDownloadHeader';
 import CustomDownloadContent from './containers/content/CustomDownloadContent';
+import CustomDownloadAppBar from './containers/app-bar/CustomDownloadAppBar';
+import { setActiveGenomeId } from './state/customDownloadActions';
+import { getCommittedSpecies } from '../species-selector/state/speciesSelectorSelectors';
+import { CommittedItem } from '../species-selector/types/species-search';
 
 type StateProps = {
   showPreFiltersPanel: boolean;
+  activeGenomeId: string | null;
+  committedItems: CommittedItem[];
 };
 
-type CustomDownloadProps = StateProps;
+type DispatchProps = {
+  setActiveGenomeId: (activeGenomeId: string) => void;
+};
+
+type CustomDownloadProps = StateProps & DispatchProps;
 
 const CustomDownload: FunctionComponent<CustomDownloadProps> = (
   props: CustomDownloadProps
 ) => {
+  useEffect(() => {
+    if (!props.activeGenomeId && props.committedItems.length) {
+      props.setActiveGenomeId(props.committedItems[0].genome_id);
+    }
+  }, [props.activeGenomeId]);
```

Comment:
Should this effect run only once, on component mount (if so, use empty array in the list of dependencies).

Reply:
That's done.